gallery/src/components/NxGlobalSidebar/NxGlobalSidebarScrollingExample.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {
+  NxGlobalSidebar,
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxGlobalSidebarFooter,
+  useToggle,
+  NxP,
+  NxH3,
+  NxPageMain,
+  NxTreeView,
+  NxCheckbox,
+  NxRadio,
+  NxTreeViewChild }
+  from '@sonatype/react-shared-components';
+import { faArrowLeft, faArrowRight, faLink, faExternalLinkSquareAlt } from '@fortawesome/free-solid-svg-icons';
+
+const logoImg = require('../../assets/images/logo-plaid-villain-text.png');
+
+export default function NxGlobalSidebarExample() {
+  const [sidebarOpen, onToggleCollapse] = useToggle(true);
+  const [is1Open, onToggle1Collapse] = useToggle(false),
+      [is2Open, onToggle2Collapse] = useToggle(false);
```

Comment:
Minor, for the sake of consistency since this is an example
```suggestion
  const [sidebarOpen, onToggleCollapse] = useToggle(true),
      [is1Open, onToggle1Collapse] = useToggle(false),
      [is2Open, onToggle2Collapse] = useToggle(false);
```